nautobot/extras/forms/mixins.py

```diff
@@ -137,6 +140,44 @@ def __init__(self, *args, **kwargs):
             self.custom_fields.append(field_name)
 
 
+class NoteFormBase:
+    """Base fore the NoteModelFormMixin and NoteModelBulkEditFormMixin."""
+
+    def append_field(self):
+        self.fields["object_note"] = CommentField()
+        self.fields["object_note"].label = "Note"
```

Comment:
Do we need this custom approach? Or can we just have:

```python
class NoteFormBase:
    """Base for the NoteModelFormMixin and NoteModelBulkEditFormMixin."""

    object_note = CommentField(label="Note")
```

Reply:
Yes, but I had to add forms.Form as the base class.